src/user_defines.txt

```diff
@@ -43,16 +43,19 @@
 # for switches.
 
 ### STEP3: ### 
-### Set your custom UID here. Use 6 random bytes! ###
--DMY_UID=0xBA,0xBE,0xCA,0xFE,0xAC,0xDC
-#-DMY_UID=180,230,45,234,66,4
+### Set your custom binding phrase here. This unique string is used to pair your ExpressLRS tx and rx. ###
+
+-DMy_Binding_Phrase="default ExpressLRS binding phrase"
```

Comment:
Pedantic: defines are typically specified in UPPERCASE to indicate that they are a define as apposed to a local variable. We already have mixed case in this file for the other defines, so feel free to ignore this if you want.